eng/common/scripts/artifact-metadata-parsing.ps1

```diff
@@ -544,3 +544,52 @@ function CheckArtifactShaAgainstTagsList($priorExistingTagList, $releaseSha, $ap
     exit(1)
   }
 }
+
+function Get-CSVMetadata ([string]$MetadataUri) {
+  $metadataResponse = Invoke-RestMethod -Uri $MetadataUri -method "GET" -MaximumRetryCount 3 -RetryIntervalSec 10 | ConvertFrom-Csv
+  return $metadataResponse
+}
+
+function Get-BlobStorage-Artifacts($blobStorageUrl, $blobDirectoryRegex) {
```

Comment:
I'd suggest we don't have a regex here we should just have the Url which can include the prefix. The regex should happen outside of this function if needed.

Reply:
I am thinking to have this function to return the package name,
```
azure-storage-queue,
azure-search-documents
```
However, if we do not have extra handling on name, for loop on the blobPrefix returns us. 
```
java\azure-storage-queue,
java\azure-search-documents
```

It is a little waste to me, if we want to make it happen outside of the function, which means we need to loop the artifacts again.